tic-tac-toe.js

```diff
@@ -1,7 +1,158 @@
 function TicTacToe() {
-
+  this.board = [[0, 0, 0], [0, 0, 0], [0, 0, 0]];
+  this.countClick = 0;
+  this.turn = 'playerOne';
+  this.one = this.board[0][0];
+  this.two = this.board[0][1];
+  this.three = this.board[0][2];
+  this.four = this.board[1][0];
+  this.five = this.board[1][1];
+  this.six = this.board[1][2];
+  this.seven = this.board[2][0];
+  this.eight = this.board[2][1];
+  this.nine = this.board[2][2];
 }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.positionAsAttribute = function(position) {
+  switch (position) {
+    case '1':
+      return this.one;
+    case '2':
+      return this.two;
+    case '3':
+      return this.three;
+    case '4':
+      return this.four;
+    case '5':
+      return this.five;
+    case '6':
+      return this.six;
+    case '7':
+      return this.seven;
+    case '8':
+      return this.eight;
+    case '9':
+      return this.nine;
+  }
+};
+
+TicTacToe.prototype.switchTurn = function () {
+  if (this.turn === 'playerOne') {
+    this.turn = 'playerTwo';
+  } else if (this.turn === 'playerTwo') {
+    this.turn = 'playerOne';
+  }
+};
```

Comment:
ternary here could be nice since it is switching between two cases consistently 
`this.turn === 'playerOne' ? 'this.turn = 'playerTwo' : this.turn = 'playerOne'`